Library/Formula/mitie.rb

```diff
@@ -14,27 +14,34 @@ class Mitie < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
+  option "with-models", "Install v0.2 models (~415MB)"
```

Comment:
I think the models are useful: they let you do things like http://blog.dlib.net/2014/07/mitie-v02-released-now-includes-python.html?m=1 out of the box. I think making them optional (`option "without-models"`) is fair though.